cfme/tests/storage/test_object_store_container.py

```diff
@@ -78,4 +56,12 @@ def test_storage_object_store_container_edit_tag_openstack():
             3. Verify the tag is assigned
             4. Verify the tag is removed
     """
-    pass
+    added_tag = container.add_tag()
+    tag_available = container.get_tags()
+    assert tag_available[0].display_name == added_tag.display_name
```

Comment:
I think its a safe assertion here, because you're verifying that the first (and presumably only) tag on the storage container is the one that was just added.

Please note that Taggable supports `in` validation, using the display_name and category display_name. `assert added_tag in container.get_tags()` would work.